Update guide - Include Examples for GPT-5 Support in OpenAI AI Model. Closes #388 (#389)
turbot/guardrails-docs: docs/guides/using-guardrails/ai/ai-configuration/index.md

The default value is set to `0.2`. You may choose to update it based on your needs.

> **Note:** Starting from OpenAI `GPT-5`, the temperature parameter is no longer supported, and any value set will be ignored by the model.

Reviewer: ```suggestion
**Note:** The temperature parameter is no longer supported for OpenAI's `GPT-5` series models; any value set for this parameter will be ignored.
```
